ravenml/utils/question.py

```diff
@@ -144,4 +144,19 @@ def cli_spinner(text, func, *args, **kwargs):
         raise
     spinner.succeed(text=text + 'Complete.')
     return result
+
```

Comment:
decorator so that class functions can be called with the spinner if wanted

Reply:
very nifty